corehq/apps/style/templates/style/includes/modal_report_issue.html

```diff
@@ -72,9 +72,23 @@ <h4 class="modal-title">{% trans "Report an Issue with CommCare HQ" %}</h4>
                         </div>
                         {% endif %}
 
+                        {% if request.couch_user.is_domain_admin %}
```

Comment:
ok offlined with robin, we are good to go to leave it the way it's currently implemented